diff --git a/libs/binder/IMemory.cpp b/libs/binder/IMemory.cpp
index c6b0cb7..5e8c3f6 100644
--- a/libs/binder/IMemory.cpp
+++ b/libs/binder/IMemory.cpp
@@ -17,7 +17,7 @@
 #define LOG_TAG "IMemory"
 
 #include <atomic>
-#include <stdatomic.h>
+// #include <stdatomic.h>
 
 #include <fcntl.h>
 #include <stdint.h>
@@ -37,6 +37,8 @@
 
 #define VERBOSE   0
 
+using namespace std;
+
 namespace android {
 // ---------------------------------------------------------------------------
 
diff --git a/libs/binder/IPCThreadState.cpp b/libs/binder/IPCThreadState.cpp
index da58251..9834c30 100644
--- a/libs/binder/IPCThreadState.cpp
+++ b/libs/binder/IPCThreadState.cpp
@@ -1162,7 +1162,7 @@ status_t IPCThreadState::talkWithDriver(bool doReceive)
             std::string message = logStream.str();
             ALOGI("%s", message.c_str());
         }
-#if defined(__ANDROID__)
+#if defined(__ANDROID__) || 1
         if (ioctl(mProcess->mDriverFD, BINDER_WRITE_READ, &bwr) >= 0)
             err = NO_ERROR;
         else
diff --git a/libs/binder/Parcel.cpp b/libs/binder/Parcel.cpp
index 0aca163..892630e 100644
--- a/libs/binder/Parcel.cpp
+++ b/libs/binder/Parcel.cpp
@@ -202,7 +202,8 @@ status_t Parcel::finishUnflattenBinder(
 
 #ifdef BINDER_WITH_KERNEL_IPC
 static constexpr inline int schedPolicyMask(int policy, int priority) {
-    return (priority & FLAT_BINDER_FLAG_PRIORITY_MASK) | ((policy & 3) << FLAT_BINDER_FLAG_SCHED_POLICY_SHIFT);
+    // return (priority & FLAT_BINDER_FLAG_PRIORITY_MASK) | ((policy & 3) << FLAT_BINDER_FLAG_SCHED_POLICY_SHIFT);
+    return 0;
 }
 #endif // BINDER_WITH_KERNEL_IPC
 
@@ -267,7 +268,7 @@ status_t Parcel::flattenBinder(const sp<IBinder>& binder) {
                 obj.flags |= FLAT_BINDER_FLAG_TXN_SECURITY_CTX;
             }
             if (local->isInheritRt()) {
-                obj.flags |= FLAT_BINDER_FLAG_INHERIT_RT;
+                // obj.flags |= FLAT_BINDER_FLAG_INHERIT_RT;
             }
             obj.hdr.type = BINDER_TYPE_BINDER;
             obj.binder = reinterpret_cast<uintptr_t>(local->getWeakRefs());
diff --git a/libs/binder/include/binder/IInterface.h b/libs/binder/include/binder/IInterface.h
index dc572ac..9f6a98e 100644
--- a/libs/binder/include/binder/IInterface.h
+++ b/libs/binder/include/binder/IInterface.h
@@ -18,7 +18,7 @@
 
 #include <binder/Binder.h>
 
-#include <assert.h>
+#include "binder_linux_assert.h"
 
 namespace android {
 
diff --git a/libs/binder/ndk/include_ndk/android/binder_status.h b/libs/binder/ndk/include_ndk/android/binder_status.h
index 76c7aac..fa468aa 100644
--- a/libs/binder/ndk/include_ndk/android/binder_status.h
+++ b/libs/binder/ndk/include_ndk/android/binder_status.h
@@ -38,6 +38,7 @@ __BEGIN_DECLS
 #define __INTRODUCED_IN(n)
 #endif
 
+#define __assert
 #ifndef __assert
 #define __assert(a, b, c)          \
     do {                           \
